web/locales/en/plugin__netobserv-plugin.json

```diff
@@ -126,9 +126,13 @@
   "Unable to get overview": "Unable to get overview",
   "No results found": "No results found",
   "Clear all filters and try again.": "Clear all filters and try again.",
+  "Also show total traffic for the selected filters": "Also show total traffic for the selected filters",
   "Show total": "Show total",
+  "Show other traffic": "Show other traffic",
   "Show others": "Show others",
+  "Show scope-internal traffic, depending on the selected scope (node-internal traffic, namespace-internal traffic)": "Show scope-internal traffic, depending on the selected scope (node-internal traffic, namespace-internal traffic)",
   "Show internal": "Show internal",
+  "Show out of scope traffic, e.g. host-network traffic when scope is not Node.": "Show out of scope traffic, e.g. host-network traffic when scope is not Node.",
```

Comment:
@memodi what do you think about those texts? Are they good enough, or would you reformulate?

Reply:
(I've changed them, cf last commit)